modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/ObjcClientCodegen.java

```diff
@@ -435,7 +435,8 @@ public boolean isDataTypeBinary(String dataType) {
     public String toModelName(String type) {
         // model name cannot use reserved keyword
         if (reservedWords.contains(type)) {
-            LOGGER.warn(type + " (reserved word) cannot be used as model name. Renamed to " + ("model_" + type) + " before further processing");
+            LOGGER.warn("{} (reserved word) cannot be used as model name. Renamed to {} before further processing",
+					type, "model_" + type);
```

Comment:
this still can be improved to `{} {}` in log message

Reply:
Interesting case, may be worth improving the cleanup.

Reply:
Good catch. We can improve it in another PR.